Jenkinsfile

```diff
@@ -93,7 +93,7 @@ OPTS+=" -Dorg.slf4j.simpleLogger.log.org.apache.maven.plugin.surefire.SurefirePl
 OPTS+=" -Dmaven.repo.local=$PWD/.git/m2"
 git config extra.mavenOpts "$OPTS"
 OPTS=" $M_OPTS -Dmaven.test.failure.ignore "
-if [ -s inclusions.txt ]; then OPTS+=" -Dsurefire.includesFile=$PWD/inclusions.txt"; sed -i '/\\/ITest/d' $PWD/inclusions.txt;fi
+if [ -s inclusions.txt ]; then OPTS+=" -Dsurefire.includesFile=$PWD/inclusions.txt";fi
```

Comment:
Just to double check: Have you checked that the tests were run by the CI?

Reply:
Yes, they all run and all passed. More details here:
```
http://ci.hive.apache.org/job/hive-precommit/job/PR-3399/1/testReport/api/xml?xpath=/testResult/suite[contains(name/text(),%22org.apache.hadoop.hive.metastore.dbinstall%22)]&wrapper=testcases
```